File: app/src/main/java/com/robotnec/budget/core/mvp/presenter/AboutPresenter.java
package com.robotnec.budget.core.mvp.presenter;

import com.robotnec.budget.core.di.ApplicationComponent;
import com.robotnec.budget.core.mvp.view.AboutView;

/**
 * @author zak <zak@swingpulse.com>
 */
public class AboutPresenter extends Presenter<AboutView> {

    public AboutPresenter(AboutView view) {
        super(view);
    }

    @Override
    public void onViewResume() {

    }

    @Override
    public void injectComponent(ApplicationComponent applicationComponent) {
        applicationComponent.inject(this);
    }
}

File: app/src/main/java/com/robotnec/budget/core/mvp/presenter/AboutPresenter.kt
package com.robotnec.budget.core.mvp.presenter

import com.robotnec.budget.core.di.ApplicationComponent
import com.robotnec.budget.core.mvp.view.AboutView

/**
 * @author zak zak@swingpulse.com>
 */
class AboutPresenter(view: AboutView) : Presenter<AboutView>(view) {

    override fun onViewResume() {

    }

    override fun injectComponent(applicationComponent: ApplicationComponent) {
        applicationComponent.inject(this)
    }
}
